src/pretix/base/exporters/orderlist.py

```diff
@@ -541,6 +542,22 @@ def iterate_positions(self, form_data: dict):
         ).order_by()
         qs = base_qs.annotate(
             payment_providers=Subquery(p_providers, output_field=CharField()),
+            checked_in_lists=Subquery(
+                Checkin.objects.filter(
+                    successful=True,
+                    type=Checkin.TYPE_ENTRY,
+                    position=OuterRef("pk"),
+                ).order_by().values("position").annotate(
+                    c=GroupConcat(
+                        "list__name",
+                        # These appear not to work properly on SQLite. Well, we don't support SQLite outside testing
+                        # anyways.
+                        ordered='sqlite' not in settings.DATABASES['default']['ENGINE'],
+                        distinct='sqlite' not in settings.DATABASES['default']['ENGINE'],
+                        delimiter=","
```

Comment:
```suggestion
                        delimiter=", "
```
statt unten zu ersetzen?

Reply:
das war der kompromiss, dass es für SQLite läuft, SQLite hat wohl nen bug im SQL parser und kann nicht DISTINCT mit custom delimiter kombinieren :angry:

Reply:
ah okay - dann könntest du aber oben drüber `distinct=True` setzen, statt nur distinct wenn die database-engine nicht sqlite ist, oder?

Reply:
m) vermutlich

Reply:
At least my SQLite is buggy, I don't get it to work with distinct, so going ro your solution